src/model/defineRelationalProperties.ts

```diff
@@ -41,6 +48,8 @@ export function defineRelationalProperties(
 
     if (references) {
       relation.resolveWith(entity, references)
+    } else if (definition instanceof NullableRelation) {
+      relation.resolveWithNull(entity)
```

Comment:
I think we should extend the `Relation` class to override its `resolveWith` method to support `null` as `references (thus preserving the call signature and getting rid of this condition). It troubles me that we'd have to treat `NullableRelation` as a special case everywhere we go. This can be eliminated by proper inheritance.

Reply:
The comment is obsolete, the `Relation` class now supports `refs` as `null`.